cpp/benchmarks/io/orc/orc_reader_input.cpp

```diff
@@ -146,3 +187,17 @@ NVBENCH_BENCH_TYPES(BM_orc_read_io_compression, NVBENCH_TYPE_AXES(io_list, compr
   .set_min_samples(4)
   .add_int64_axis("cardinality", {0, 1000})
   .add_int64_axis("run_length", {1, 32});
+
+std::size_t constexpr Mbytes{1024 * 1024};
+
+// Should have the same parameters as `BM_orc_read_io_compression` for comparison.
+NVBENCH_BENCH_TYPES(BM_orc_chunked_read_io_compression,
+                    NVBENCH_TYPE_AXES(io_list, compression_list))
+  .set_name("orc_chunked_read_io_compression")
+  .set_type_axes_names({"io", "compression"})
+  .set_min_samples(4)
+  .add_int64_axis("cardinality", {0, 1000})
+  .add_int64_axis("run_length", {1, 32})
```

Comment:
Are we getting a different signal about the impact of chunking with different cardinality/run length?
I'd instead rather cover a larger variety of the number of chunks, if possible.

Reply:
Then we can fix `cardinality` and `run_length`. What values should be fixed? How about using just the first values from that pair?

Reply:
I'm leaning towards 0 cardinality and ~4 RL. Would also be fine with a relatively high non-zero cardinality.